csharp/ql/lib/semmle/code/csharp/dataflow/internal/FlowSummaryImplSpecific.qll

```diff
@@ -114,6 +114,18 @@ predicate summaryElement(Callable c, string input, string output, string kind, b
   )
 }
 
+/**
+ * Holds if an external flow summary exists for `c` which means that there is no
+ * flow through `c` and a flag `generated` stating whether the summary is autogenerated.
```

Comment:
There's a grammatical mistake here that really messes with the meaning of the sentence.
```suggestion
 * Holds if a negative flow summary exists for `c`, which means that there is no
 * flow through `c`. The flag `generated` states whether the summary is autogenerated.
```

Reply:
Yes - I will fix it.